common/script_values/00_men_at_arms_values.txt

```diff
@@ -67,6 +67,12 @@ heavy_cavalry_recruitment_cost = @[heavy_cavalry_recruitment_cost]
 heavy_cavalry_low_maint_cost = @[heavy_cavalry_low_maint_cost]
 heavy_cavalry_high_maint_cost = @[heavy_cavalry_high_maint_cost]
 
+#Warcraft
+
+riflemen_recruitment_cost = 120
+riflemen_low_maint_cost = @[archers_low_maint_cost * 1.8]
```

Comment:
Added to the docs way to calculate maint cost so we don't use archers.
https://docs.google.com/spreadsheets/d/18zS-FAte6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=583000646&range=H186:H189